llmfoundry/models/hf/hf_causal_lm.py

```diff
@@ -135,6 +136,17 @@ def __init__(
             should_save_peft_only=should_save_peft_only,
         )
 
+    def transform_model(self, model: PreTrainedModel) -> PreTrainedModel:
```

Comment:
this fn will optionally transform model, including peft adapters right

Reply:
transform is called before the peft config is created